.github/workflows/tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        resolver: [nightly]
```

Comment:
Please don't drop Stackage Nightly from the list of supported build environments. It is important that we know our code builds in the Nightly snapshot because that environment is going to turn into the next LTS release, eventually.

Reply:
This is a good point and I'm sorry for not noticing this in #266. I think it could've been addressed in a followup commit and didn't need a revert, though.

Reply:
My plan to include new lts as it's created. But that's a good point and I will update the MR to include that too.